stories/tableOfContents/listItems.json

```diff
@@ -1,13 +1,13 @@
 [
-  {"text": "Purpose", "active": true},
-  {"text": "Objectives", "active": false},
-  {"text": "Mandate", "active": false},
-  {"text": "Scope", "active": false},
-  {"text": "Challenges", "active": false},
-  {"text": "Principles", "active": false},
-  {"text": "Categories of Commitment", "active": false},
-  {"text": "Frequency of Policy Review", "active": false},
-  {"text": "Roles and Responsibilities", "active": false},
-  {"text": "Appendix A: Glossary", "active": false},
-  {"text": "Appendix B: Sources Consulted", "active": false}
+  {"text": "Purpose"},
+  {"text": "Objectives"},
+  {"text": "Mandate"},
+  {"text": "Scope"},
+  {"text": "Challenges"},
+  {"text": "Principles"},
+  {"text": "Categories of Commitment"},
+  {"text": "Frequency of Policy Review"},
+  {"text": "Roles and Responsibilities"},
+  {"text": "Appendix A: Glossary"},
+  {"text": "Appendix B: Sources Consulted"}
```

Comment:
Can you explain more about removing the `active` class and then adding a style for `a:active` below?

Reply:
I was thinking that the `:active` pseudo-class should cover it without it needing to be its own class. Did I overlook the reason for that approach?

Reply:
Hmm I am wondering if we added an `.active` class so we could style the ToC item that is visible on the page. There's some styling like this on the docs site...